### ISSUE TITLE
line number update slow

### ISSUE THREAD

**Issue by xuuexu** [redguardtoo/emacs.d#681]
Sir, I lately have tried your config.
There's one thing that's not good for me is the speed of updating line number.
It seem to update the line number when I am back to Normal Mode and wait about 1 second.
Is it a feature or something buggy?
 
OS: MacOS 10.13.4
GUI: Emacs-plus 25.3

**Comment by redguardtoo** [comment]
Yes, it's a feature. Or else Emacs general performance is drag down by `linum-mode`. 

See https://lists.gnu.org/archive/html/bug-gnu-emacs/2013-04/msg00577.html
